spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkDataWrite.java

```diff
@@ -71,9 +72,18 @@ public class TestSparkDataWrite {
 
   @Rule public TemporaryFolder temp = new TemporaryFolder();
 
-  @Parameterized.Parameters(name = "format = {0}")
+  private String targetBranch;
```

Comment:
This test looks good, but I think we need to find the other paths that are affected and update those tests as well.